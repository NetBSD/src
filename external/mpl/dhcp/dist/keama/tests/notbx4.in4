# not boolean expression

# empty configs are not accepted by Kea
default-lease-time 1800;

# use not in a reductible match if
class "reductible" {
    match if not (option host-name = "www.example.com");
}

# if test is a boolean too
if not check "foo" { add "bar"; }
