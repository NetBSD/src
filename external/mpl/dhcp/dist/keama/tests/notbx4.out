{
  # not boolean expression
  # empty configs are not accepted by Kea
  "Dhcp4": {
    "valid-lifetime": 1800,
    "client-classes": [
      # use not in a reductible match if
      {
        "name": "reductible",
        /// from: match if not (option dhcp.host-name) = 'www.example.com'
        "test": "not (option[12].hex == 'www.example.com')"
      }
    ]
//  # if test is a boolean too
//  "statement": {
//    "if": {
//      "condition": {
//        "not": {
//          "check": "foo"
//        }
//      },
//      "then": [
//        {
//          "add-class": "bar"
//        }
//      ]
//    }
//  }
  }
}
